src/gui/folderwizard.cpp

```diff
@@ -523,6 +526,31 @@ void FolderWizardRemotePath::showWarn(const QString &msg) const
     }
 }
 
+void FolderWizardRemotePath::changeEvent(QEvent *e)
+{
+    switch (e->type()) {
+    case QEvent::StyleChange:
+    case QEvent::PaletteChange:
+    case QEvent::ThemeChange:
+        // Notify the other widgets (Dark-/Light-Mode switching)
+        changeStyle();
+        break;
+    default:
+        break;
+    }
+
+    FormatWarningsWizardPage::changeEvent(e);
+}
+
+void FolderWizardRemotePath::changeStyle()
+{
+    const auto warnYellow = Theme::instance()->darkMode() ? QColor(63, 63, 0) : QColor(255, 255, 192);
```

Comment:
@claucambra the body seems identical to `FolderWizardLocalPath::changeStyle()`, could you aply DRY? If not possible, at least move colors to `constexpr`

Reply:
I moved the palette generation to a shared function in anonymous namespace, unfortunately can't move everything out as the `_ui` types are different